static/app.js

```
@@ -0,0 +1,228 @@
+        render_message_history();
+    };
+
+    socket.onclose = () => {
+        console.log('WebSocket connection closed');
+    };
+}
+
+function sendMessage() {
+    const messageInput = document.getElementById('messageInput');
+    const message = messageInput.value;
+    let message_type = "text"
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'message_type' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)